spec/std/crystal/compiler_rt/multi3_spec.cr

```diff
@@ -0,0 +1,63 @@
+require "spec"
+require "crystal/compiler_rt/multi3"
+
+# Ported from https://github.com/llvm/llvm-project/blob/ce59ccd04023cab3a837da14079ca2dcbfebb70c/compiler-rt/test/builtins/Unit/multi3_test.c
+
+private def test__mulodi4(a : Int64, b : Int64, expected : Int64, expected_overflow : Int32, file = __FILE__, line = __LINE__)
+  it "passes compiler-rt builtins unit tests" do
+    actual_overflow : Int32 = 0
+    actual = __mulodi4(a, b, pointerof(actual_overflow))
+    actual_overflow.should eq(expected_overflow), file: file, line: line
+    if !expected_overflow
+      actual.should eq(expected), file: file, line: line
+    end
+  end
+end
```

Comment:
unused helper (`test__multi3` has only a customized error message so it is fine if the helper is dropped altogether)

Reply:
Yeah, that was just a leftover.